test/sortitionPoolTest.js

```diff
@@ -436,4 +528,142 @@ describe("SortitionPool", () => {
       expect(group.length).to.equal(100)
     })
   })
+
+  describe("addBetaOperators", () => {
+    context("when called by third party", () => {
+      it("should revert", async () => {
+        await expect(
+          pool.connect(thirdParty).addBetaOperators([alice.address]),
+        ).to.be.revertedWith("Not the chaosnet owner")
+      })
+    })
+
+    context("when called by the operator", () => {
```

Comment:
`alice` is not a beta operator here. It duplicates the scenario above, where caller is a `thirdParty`

Reply:
The intention of this test was to check that the operator can not add itself as a beta operator. So yeah, it's really similar to  the third party but it's not the same. Third-party wants to add someone else but `alice` wants to add `alice`.

Reply:
ok, so IMO the context description is a bit misleading. It says "when called by the operator", but here `alice` is not an operator, but some random account just like `thirdParty`.

Reply:
`addBetaOperators` that is called in this test accepts `address[] calldata operators` and I named `alice` an operator because she is one of the elements of the passed array. 

How can we make it more clear? Change the test description, add a comment?

Reply:
My point is that when I read "when called by the operator", I look at this part `pool.connect(alice)` and not at this `.addBetaOperators([alice.address])`. Here, `alice` is an operator wannabe, not yet an operator, that's why it looked duplicate to me with the scenario above. I would say "when called by self" or smth similar. But this is really minor, if you want to leave it as-is I'm okay with that.

Reply:
Let's see what @nkuba says about it then.

Reply:
I would leave this as is, this is a common naming pattern we use in our tests.